charts/fission-all/templates/fluentd.yaml

```diff
@@ -0,0 +1,81 @@
+# Fluentd deployment for Fission
+#
+# Requires:
+# - service account: fission-svc
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-fission-fluentd
+data:
+{{ if .Files.Get "config/fluentd.conf" }}
+  td-agent.conf: |
+{{ .Files.Get "config/fluentd.conf" | indent 3 }}
+{{ else }}
+{{ fail "you need to supply a fluentd config file! add 'fluentd.conf' to your chart 'config' directory." }}
```

Comment:
Just "invalid chart".  User isn't going to be able to fix this by themselves, if it happens it's simply a bug in our chart.